README.md

```diff
@@ -78,13 +151,24 @@ I would like to thank Microverse for this wonderful journey.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
```

Comment:
- [Optional] Kindly take this section out. And always remember to do so for all future projects. all lines with the `>` sections should serve as a guide but not in the project